README.md

```diff
@@ -9,6 +9,7 @@ This repository is the home of the following packages:
 -   [`@eslint/object-schema`](packages/object-schema)
 -   [`@eslint/config-array`](packages/config-array)
 -   [`@eslint/compat`](packages/compat)
+-   [`@eslint/migrate-config`](packages/migrate-config)
```

Comment:
lets add the same at https://github.com/eslint/rewrite/blob/9f62bbc0bfed68c9b4c639c5f88db3c31b5d70ac/.github/ISSUE_TEMPLATE/bug-report.yml?plain=1#L18

Reply:
and here https://github.com/eslint/rewrite/blob/9f62bbc0bfed68c9b4c639c5f88db3c31b5d70ac/.github/ISSUE_TEMPLATE/change.yml?plain=1#L17

Reply:
ah nvm my bad its already there at both places